### PR TITLE
navigator.serviceWorker may throw an exception due to the sandbox settings of the iframe (fix #277)

### LINKED ISSUE #277: navigator.serviceWorker may throw an exception due to the sandbox settings of the iframe
```javascript
test('', function() {
    var iframe = document.createElement('iframe');

    iframe.setAttribute('sandbox', '');
    iframe.src = window.getCrossDomainPageUrl('../../../data/cross-domain/service-worker-not-available.html');

    document.body.appendChild(iframe);
});
```
```Uncaught SecurityError: Failed to read the 'serviceWorker' property from 'Navigator': Service worker is disabled because the context is sandboxed and lacks the 'allow-same-origin' flag.```
[Source code](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/native-methods.js#L96) where the error occurs.

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

File: src/client/sandbox/native-methods.js [hunk 1]
```diff
@@ -93,8 +93,14 @@ class NativeMethods {
         this.setInterval             = win.setInterval || Window.prototype.setInterval;
         this.XMLHttpRequest          = win.XMLHttpRequest;
         this.registerProtocolHandler = win.navigator.registerProtocolHandler;
-        this.registerServiceWorker   = win.navigator &&
-                                       win.navigator.serviceWorker ? win.navigator.serviceWorker.register : null;
+
+        try {
+            this.registerServiceWorker = win.navigator && win.navigator.serviceWorker
+                ? win.navigator.serviceWorker.register : null;
```
Comment (inikulin) [inline src/client/sandbox/native-methods.js]: Bad formatting of the ternary operator. I'd rather keep it one-liner

**Comment by inikulin** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

**Comment by inikulin** [comment]
Lgtm

**Comment by miherlosev** [comment]
lgtm

**Pull request merged**